### ISSUE TITLE
[Bug]: Game 'Genshin impact' 5.2 Version Unable to connect to network4201

### ISSUE THREAD

**Issue by lanzaizai816** [PlayCover/PlayCover#1743]
### Describe the bug

When I opened the game, it was immediately shown that I couldn't connect to the Internet 4201
<img width="1318" alt="Screenshot 2024-11-20 at 12 43 20" src="https://github.com/user-attachments/assets/9dfd8eb0-be5d-4dad-8345-d21cf3b5848f">

### Steps to reproduce

When I opened the game, it was immediately shown that I couldn't connect to the Internet, and trying to install the latest version didn't help

### Expected behaviour

genshin impact this game has a network connection problem

### What version of PlayCover are you using?

Nightly/beta (please specify build number)

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [x] Yes my issue is written in English

**Comment by JiuZX** [comment]
Please delete the mihoyo address in the `/etc/hosts` what you add before.

**Comment by Yew0329** [comment]
我也遇到了这个问题，请问有什么办法能解决？？？上面的回答我并没有理解

**Comment by lanzaizai816** [comment]
> Please delete the mihoyo address in the `/etc/hosts` what you add before.

How do I delete the hosts file on my mac computer?

**Comment by Yew0329** [comment]
是的，我没有理解前面的那一步操作如何运行，能否给出具体一点的指导

**Comment by JiuZX** [comment]
Just edit the file like you add it. Run `sudo vim /etc/hosts` to edit the file, and then delete the line, save the files.

**Comment by lanzaizai816** [comment]
> 我也遇到了这个问题，请问有什么办法能解决？？？上面的回答我并没有理解

应该是之前闪退修改hosts之后，导致现在进不去游戏了，要把之前添加的hosts删除保存，还在尝试

**Comment by lanzaizai816** [comment]
> Just edit the file like you add it. Run `sudo vim /etc/hosts` to edit the file, and then delete the line, save the files.

Okay, thanks. I'll keep trying

**Comment by Yew0329** [comment]
等等，原谅我的电脑技术非常一窍不通，我仍未理解你们说的那个主机保存或者文件是具体指哪一个？可以在访达搜关键词找到吗

**Comment by JiuZX** [comment]
Right. You need to revert the change in https://github.com/PlayCover/PlayCover/issues/1672#issuecomment-2343617972

**Comment by lanzaizai816** [comment]
> 等等，原谅我的电脑技术非常一窍不通，我仍未理解你们说的那个主机保存或者文件是具体指哪一个？可以在访达搜关键词找到吗
按shift+command+G快捷键打开搜索，输入/etc/打开文件夹找到hosts这个文件复制备份并删除，就可以进入下载了，但不知道下载完还能不能正常启动，还在下载中
<img width="896" alt="截屏2024-11-20 13 56 35" src="https://github.com/user-attachments/assets/c8c31f10-8967-4eb0-bae6-c050b23ee013">
<img width="717" alt="截屏2024-11-20 13 57 25" src="https://github.com/user-attachments/assets/b6d34e02-598d-4be0-946a-42b50286dd1f">
<img width="727" alt="截屏2024-11-20 13 57 47" src="https://github.com/user-attachments/assets/92263e1e-7fde-4393-a45d-c7da0a5e756c">

**Comment by Yew0329** [comment]
你们是正确的，感谢你们的付出，问题已得到解决

**Comment by Psinary** [comment]
问一下你们是在哪儿找的的砸壳的IPA的，又什么渠道吗？

**Comment by Yew0329** [comment]
有几个网站有免费分享，每次更新的时候就去下载就好了



发自我的iPhone


------------------ Original ------------------
From: 雷电国崩 ***@***.***&gt;
Date: Wed,Nov 20,2024 3:00 PM
To: PlayCover/PlayCover ***@***.***&gt;
Cc: Yew0329 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [PlayCover/PlayCover] [Bug]: Game 'Genshin impact' 5.2 VersionUnable to connect to network4201 (Issue #1743)





 
问一下你们是在哪儿找的的砸壳的IPA的，又什么渠道吗？

**Comment by Psinary** [comment]
> 有几个网站有免费分享，每次更新的时候就去下载就好了 发自我的iPhone
> […](#)
> ------------------ Original ------------------ From: 雷电国崩 ***@***.***&gt; Date: Wed,Nov 20,2024 3:00 PM To: PlayCover/PlayCover ***@***.***&gt; Cc: Yew0329 ***@***.***&gt;, Comment ***@***.***&gt; Subject: Re: [PlayCover/PlayCover] [Bug]: Game 'Genshin impact' 5.2 VersionUnable to connect to network4201 (Issue #1743) 问一下你们是在哪儿找的的砸壳的IPA的，又什么渠道吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >又推荐的吗？求一份。

**Comment by Yew0329** [comment]
例如：


https://ipa.store/这个在国内比较快一点


https://decrypt.day/这个网站在外面有时候需要加速器

发自我的iPhone


------------------ Original ------------------
From: 雷电国崩 ***@***.***&gt;
Date: Wed,Nov 20,2024 3:13 PM
To: PlayCover/PlayCover ***@***.***&gt;
Cc: Yew0329 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [PlayCover/PlayCover] [Bug]: Game 'Genshin impact' 5.2 VersionUnable to connect to network4201 (Issue #1743)





  
有几个网站有免费分享，每次更新的时候就去下载就好了 发自我的iPhone
 …
 ------------------ Original ------------------ From: 雷电国崩 @.&gt; Date: Wed,Nov 20,2024 3:00 PM To: PlayCover/PlayCover @.&gt; Cc: Yew0329 @.&gt;, Comment @.&gt; Subject: Re: [PlayCover/PlayCover] [Bug]: Game 'Genshin impact' 5.2 VersionUnable to connect to network4201 (Issue #1743) 问一下你们是在哪儿找的的砸壳的IPA的，又什么渠道吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: @.***&gt;
 又推荐的吗？求一份。

**Comment by Psinary** [comment]
好的，非常感谢。<img src="https://img-static.mihoyo.com/communityweb/upload/1591742ff2e0b8954f870efec48de98f.png" alt="烟绯-得意" width="75" >

**Comment by Psinary** [comment]
贴个解决命令
>sudo sed -i '' -e '/0.0.0.0 dispatchcnglobal.yuanshen.com/d' -e '/0.0.0.0 ys-log-upload.mihoyo.com/d' -e '/0.0.0.0 apm-log-upload-v6.mihoyo.com/d' /etc/hosts

<img src="https://bbs-static.miyoushe.com/static/2024/10/28/8129c493cacb619cbb59bed6bf59133c_2891051155203114632.png" alt="纳西妲-幸福" width="75" >
